src/components/burger-constructor/burger-constructor.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import styles from "./burger-constructor.module.css";
+import { ConstructorElement, DragIcon, CurrencyIcon, Button } from "@ya.praktikum/react-developer-burger-ui-components";
+import PropTypes from "prop-types";
+
+const BurgerConstructor = ({ data }) => {
+  return (
+    <section className={styles.container}>
+      <div className={styles.locked}>
+        <ul className={styles.list}>
+          <li className={styles.element + " mb-4 mr-2 ml-4 pl-8"} key={data[0]._id}>
```

Comment:
У булок не нужно задавать атрибут key, так как реакт в консоли выдаёт предупреждение:

> Encountered two children with the same key, `60666c42cc7b410027a1a9b1`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

Reply:
Добрый день! Спасибо большое Вам за комментарии и замечания, внес правки и сделал коммит